### PR TITLE
added more steps

### PR DESCRIPTION
**Pull request by AlexGalichenko**
added select option step
added cookie steps
added local/session storage steps

### CODE REVIEW

File: src/actions.ts [hunk 1]
```diff
@@ -158,3 +158,39 @@ When('I hover over {string}', async function (alias: string) {
...
+    await conditionWait(select, conditionValidations.VISIBLE, config.browser.timeout.visible);
+    await select.selectByIndex(optionIndex)
+});
+
+/**
+ * Save page screenshot into memory
+ * @param {string} key - key to store value
+ * @example I save screenshot as 'screenshot'
+ */
+When('I save screenshot as {string}', async function(key: string) {
```
[the remark this answers is not in the document]
Comment (AlexGalichenko) [reply src/actions.ts:193]: Actually, I plan to make comparison module agnostic to where we take screenshot, so I think we have to create such step in UI-related modules and then user screenshot stored in memory for comparison testing

File: src/actions.ts [hunk 2]
```diff
@@ -158,3 +158,39 @@ When('I hover over {string}', async function (alias: string) {
...
 });
+
+/**
+ * Select option with certain text from select element
+ * @param {string} option - option to select
+ * @param {string} alias - alias of select
+ * @example I select '1900' option from 'Registration Form > Date Of Birth'
+ * @example I select '$dateOfBirth' option from 'Registration Form > Date Of Birth'
+ */
+When('I select {string} option from {string} select', async function (option: string, alias: string) {
```
Comment (nomid0612) [inline src/actions.ts:169]: maybe it is better to rename select to dropdown

Comment (AlexGalichenko) [reply src/actions.ts:169]: make sense

Comment (AlexGalichenko) [reply src/actions.ts]: updated

**Pull request merged**